.github/workflows/main.yml

```diff
@@ -15,16 +15,16 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: 8
-      - name: Install Maven Wrapper
+      - name: Install Maven
```

Comment:
Let's either remove this here, or add it to all of the jobs. The reason we needed it here before was that the `docker/build-local.sh` script uses it, but fails if Maven isn't already downloaded, as the download process writes its status information to stdout.

I'd prefer to remove it to keep things less verbose.

Reply:
That's not a code style change, but will remove